test_sets/Ideas_for_SDL_must_send_WARNINGS_success_true_to_mobile_app_if_HMI_respond_WARNINGS_at_least_to_one_component_of_RPC.txt

```diff
@@ -0,0 +1,29 @@
+./test_scripts/API/SetGlobalProperties/ATF_SetGlobalProperties_TTS_SetGlobalProperties_WARNINGS.lua Checks_that_SDL_sends_WARNINGS_(success_true)_to_mobile_app_in_case_HMI_respond_WARNINGS_to_TTS_SetGlobalProperties_UI_SetGlobalProperties_SUCCESS

```

Comment:
@vvvakulenko, please update BA for the mistake in requirement to be updated also.
Agree for TS.